aws-fluentd-cell.conf

```diff
@@ -13,10 +13,6 @@
 </source>
 
 <label @containers>
-  <filter **>
-    @type kubernetes_metadata
-    @id filter_kube_metadata
-  </filter>
```

Comment:
Let's comment it out for now, and look into adding it back. It'd be nice to have metadata in logs.